ethexe/observer/src/lib.rs

```diff
@@ -64,15 +64,24 @@ pub struct EthereumConfig {
     pub block_time: Duration,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct BlockSyncedData {
```

Comment:
more likely BlockAuthorities, wdyt?

Reply:
also please place todo as we agreed to cache and operate with validators more accurate

Reply:
I do not think BlockAuthorities is a good name, because this is abstract data which is attached to block synced events. something else also could be added here or removed

Reply:
> also please place todo as we agreed to cache and operate with validators more accurate

added in service/lib.rs